js/app.js

```diff
@@ -8,32 +22,56 @@ const STATE = {
       tiles: ["tile2"],
     },
     arrays: {
-      tiles: ["tile2", "tile3"],
+      tiles: ["tile3"],
+    },
+    javascript: {
+      tiles: ["tile2"],
     },
   },
   exceptions: ["to", "the", "best"],
   tiles: {
     tile1: {
       id: 1,
```

Comment:
As we discussed previously, if you add the id inside the **object value**, in this case the **id**, it should have the same value as the **object key**, so if you have a _key_ with value **tile1** the _id_ value should be **title1** and not **1**